### ISSUE TITLE
Expose all arbitraries supported by `fc.object` as their own arbitraries

### ISSUE THREAD

**Issue by TomerAberbach** [dubzzz/fast-check#5499]
## 🚀 Feature Request

[`fc.object`](https://fast-check.dev/docs/core-blocks/arbitraries/composites/object/#object-1) supports emitting all sorts of stuff for its values.

Some of them have their own arbitraries (e.g. `withBigInt` option corresponds to `fc.bigInt`), but some do not (e.g. `withMap` option doesn't have a corresponding exported arbitrary).

I think all of the things that `fc.object` can generate should be supported as standalone arbitraries too.

## Motivation

It's convenient to be able to generate these types outside an object.

## Example

```js
// To generate Map<string, unknown>
fc.map(fc.string(), fc.anything())
```

**Comment by dubzzz** [comment]
Good idea 👍 I think my main blocker was `set` as it was something in v2. But with v2 I don't have it anymore so I could clearly introduce these helpers.

**Comment by dubzzz** [comment]
I have it in my to-do. Mostly focusing on having v4 released first. Then I have plenty of minors to work on with small features to enrich our feature set like this one.

**Comment by dubzzz** [comment]
On it, I started to work on it for `map`: https://github.com/dubzzz/fast-check/pull/6267.
But also for `set`: https://github.com/dubzzz/fast-check/pull/6270.

**Comment by dubzzz** [comment]
Just released the set and map on npm 🚀

**Comment by TomerAberbach** [comment]
Very nice :)
